tests/CodeBehind/BuildTests/Resources/layout-land/onboarding_intro.xml

```diff
@@ -3,16 +3,16 @@
 	      android:id="@+id/onboarding_intro">
     <TextView android:id="@+id/welcome" />
     <RelativeLayout>
-	<RelativeLayout android:id="@+id/onboarding_info">
```

Comment:
This "inconsistency" is *deliberate*; see: https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/LayoutCodeBehind.md#processing